float64 scale_by

geometry_msgs/PoseStamped commanded_pose

bool in_navigation_mode

geometry_msgs/PoseStamped end_effector_pose

